libraries/protocol/include/graphene/protocol/restriction.hpp

```diff
@@ -59,19 +59,19 @@ struct restriction {
    /*  3 */ string, \
    /*  4 */ time_point_sec, \
    /*  5 */ public_key_type, \
-   /*  6 */ fc::sha256, \
-   /*  7 */ account_id_type, \
-   /*  8 */ asset_id_type, \
-   /*  9 */ force_settlement_id_type, \
-   /* 10 */ committee_member_id_type, \
-   /* 11 */ witness_id_type, \
-   /* 12 */ limit_order_id_type, \
-   /* 13 */ call_order_id_type, \
-   /* 14 */ custom_id_type, \
-   /* 15 */ proposal_id_type, \
-   /* 16 */ withdraw_permission_id_type, \
-   /* 17 */ vesting_balance_id_type, \
-   /* 18 */ worker_id_type, \
+   /*  6 */ fc::sha1, \
+   /*  7 */ fc::sha256, \
+   /*  8 */ fc::ripemd160, \
+   /*  9 */ fc::hash160, \
+   /* 10 */ account_id_type, \
```

Comment:
Unfortunately the reordering breaks the public testnet.

Reply:
Ahh? I didn't realize this had been released, much less used. The fix is just to move the new hash types to the bottom of the list, no?